### ISSUE TITLE
Contour Fill ignores end position

### ISSUE THREAD

**Issue by bianchifan** [inkstitch/inkstitch#1903]
All fillstitch version accept start and end positions except the contour fill.
Contourfill always starts AND ends at the start position.
![contourfill](https://user-images.githubusercontent.com/17491743/201071733-0686df30-b0bf-4d5c-a06a-66e02010bfbe.jpg)

**Comment by lexelby** [comment]
Yup, that's a design choice we made when we created contour fill.  Maybe we'll add support for the end position in the future.  I'm not quite sure what it should look like though...

**Comment by kaalleen** [comment]
Hm, I wonder if that is very clever though. Running with a straight line to the start point again while crossing the top stitch layer seems like unwanted behaviour?

**Comment by kaalleen** [comment]
Maybe it could simply travel around the outline to end up at the end position?
This could be troublesome though if the end position lies inside a hole...
